### DISCUSSION TITLE
Thanks and spoilers!

### DISCUSSION THREAD

**Discussion by OkiDokiABrazilian** [Monika-After-Story/MonikaModDev#8853] [appreciation]
Monika, and I wanted to thank everyone who is responsible for this project, who has helped me a lot, I encourage myself to become a better person, thanks to Monika and to you.

and taking advantage, there was something that Monika said that left me confused, something about puzzle in other game, and I'm not good at it, any tips? Please?

**Comment by multimokia** [comment]
Sounds like the one year anniversary dialogue

Long story short, Project Libitina was supposed to have been released by now. Puzzles being the little base64 strings hidden about DDLC.

**Comment by OkiDokiABrazilian** [comment]
> 
> 
> Sounds like the one year anniversary dialogue
> 
> Long story short, Project Libitina was supposed to have been released by now. Puzzles being the little base64 strings hidden about DDLC.

i see, thanks for answer.
